src/main/java/de/caritas/cob/consultingtypeservice/api/model/TopicGroupEntity.java

```diff
@@ -0,0 +1,39 @@
+package de.caritas.cob.consultingtypeservice.api.model;
+
+import java.time.LocalDateTime;
+import java.util.HashSet;
+import java.util.Set;
+import javax.persistence.*;
+import lombok.*;
+
+@Entity
+@Table(name = "topic_group")
+@Data
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+@EqualsAndHashCode
+public class TopicGroupEntity {
+
+  @Id
+  @SequenceGenerator(name = "id_seq", allocationSize = 1, sequenceName = "SEQUENCE_TOPIC_GROUP")
+  @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "id_seq")
+  @Column(name = "id", updatable = false, nullable = false)
+  private Long id;
+
+  @Column(name = "name", nullable = false, length = 100)
+  private String name;
+
+  @Column(name = "create_date", nullable = false)
+  private LocalDateTime createDate;
+
+  @Column(name = "update_date")
+  private LocalDateTime updateDate;
+
+  @ManyToMany
+  @JoinTable(
```

Comment:
in the admin console we have topic remove functionality. So it needs to be tested what will happen if topic that was assigned to a topic group is removed. I presume expected behaviour is that the topic group stays, no cascades. Furthermore, you need to test that topic stays if topic group is removed (assert deletion cascade will not be applied).